readthedocs/proxito/tests/test_headers.py

```diff
@@ -38,6 +38,22 @@ def test_serve_headers(self):
         self.assertEqual(r['X-RTD-version-Method'], 'path')
         self.assertEqual(r['X-RTD-Path'], '/proxito/media/html/project/latest/index.html')
 
+    def test_subproject_serve_headers(self):
+        r = self.client.get('/projects/subproject/en/latest/', HTTP_HOST='project.dev.readthedocs.io')
+        self.assertEqual(r.status_code, 200)
+        self.assertEqual(r['Cache-Tag'], 'subproject,subproject-latest')
+        self.assertEqual(r['X-RTD-Domain'], 'project.dev.readthedocs.io')
+        self.assertEqual(r['X-RTD-Project'], 'subproject')
+
+        # I think it's not accurate saying that it's `subdomain` the method
+        # that we use to get the project slug here, since it was in fact the
+        # URL's path but we don't have that feature built
+        self.assertEqual(r['X-RTD-Project-Method'], 'subdomain')
```

Comment:
This is not related to this PR, but I wanted to make a note here, tho. Maybe a TODO where we set this header is better? https://github.com/readthedocs/readthedocs.org/blob/a81d82ed736b611491050f39649bd668ef212c0f/readthedocs/proxito/middleware.py#L141-L146

This header is wrong to all the subprojects when they are accessed via the domain of their parent's, eg. `project.readthedocs.io/projects/subproject/en/latest/`